[Issue]: rocWMMA is not integrated into python packages

### Problem Description

I've been testing/building llama.cpp with rocWMMA and noticed that the rocWMMA header isn’t included when building Python packages via `build_python_packages.py`. I double checked that there is built rocWMMA in `build` directory of ROCm and that there is no rocWMMA in `_rocm_sdk_devel`. There is a [discussion on Discord](https://discord.com/channels/1239631572886491286/1438824822531424276) regarding this issue.


<details>
<summary><b>Click to reveal Detailed error log</b></summary>

```bash
[1/303] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-dw.cu.o
FAILED: ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-dw.cu.o
/therock/src/.venv/lib/python3.12/site-packages/_rocm_sdk_devel/llvm/bin/clang -DGGML_BACKEND_BUILD -DGGML_BACKEND_SHARED -DGGML_HIP_NO_VMM -DGGML_HIP_ROCWMMA_FATTN -DGGML_SCHED_MAX_COPIES=4 -DGGML_SHARED -DGG
ML_USE_HIP -DUSE_PROF_API=1 -D_GNU_SOURCE -D_XOPEN_SOURCE=600 -D__HIP_PLATFORM_AMD__=1 -D__HIP_ROCclr__=1 -Dggml_hip_EXPORTS -I/therock/src/external-builds/llama.cpp/llama.cpp/ggml/src/ggml-hip/.. -I/therock/s
rc/external-builds/llama.cpp/llama.cpp/ggml/src/../include --gpu-max-threads-per-block=1024 -O3 -DNDEBUG -std=gnu++17 --offload-arch=gfx1100 -fPIC -MD -MT ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda
/conv2d-dw.cu.o -MF ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-dw.cu.o.d -o ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-dw.cu.o -x hip -c /therock/src/external-builds/llama
.cpp/llama.cpp/ggml/src/ggml-cuda/conv2d-dw.cu
In file included from /therock/src/external-builds/llama.cpp/llama.cpp/ggml/src/ggml-cuda/conv2d-dw.cu:1:
In file included from /therock/src/external-builds/llama.cpp/llama.cpp/ggml/src/ggml-cuda/conv2d-dw.cuh:2:
In file included from /therock/src/external-builds/llama.cpp/llama.cpp/ggml/src/ggml-cuda/common.cuh:31:
/therock/src/external-builds/llama.cpp/llama.cpp/ggml/src/ggml-cuda/vendors/hip.h:10:10: fatal error: 'rocwmma/rocwmma-version.hpp' file not found
   10 | #include <rocwmma/rocwmma-version.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated when compiling for gfx1100.
```
</details>



### Operating System

Manylinux 

### CPU

AMD EPYC 7453

### GPU

AMD Radeon RX 7900XTX

### ROCm Version

7.10.0

### ROCm Component

rocWMMA

### Steps to Reproduce

To reproduce the error, build ROCm with TheRock and then build llama.cpp with WMMA support enabled. You can use scripts from here #1652, and run:
1. `python llamacpp_repo.py`
2. `python llamacpp_build.py`

It works perfectly fine when you specify direct path to the built ROCm via TheRock:
```bash
python llamacpp_build.py --rocm-targets gfx1100 --rocm-dir /therock/output/build/dist/rocm/
```


### (Optional for Linux users) Output of /opt/rocm/bin/rocminfo --support

_No response_

### Additional Information

_No response_

--- a/build_tools/_therock_utils/py_packaging.py
+++ b/build_tools/_therock_utils/py_packaging.py
@@ -349,7 +349,7 @@ class PopulatedDistPackage:
             ]
             subprocess.check_call(patchelf_cl)
 
-      def _normalize_rpath(self, file_path: Path):
+    def _normalize_rpath(self, file_path: Path):
         existing_rpath = (
             subprocess.check_output(
                 [

